### PR TITLE
Add checkmark after selected button

### LINKED ISSUE #234: Contrast on selected vote is too low
image

(Blank is selected, Ja is hovered)

Looks fine on my screen, but we should probably bump up the contrast a bit.

### PR DESCRIPTION
**Pull request by Letaguator**
Fixes #234.
Add checkbox after selected button, to indicate that it is selected, therefore montitors with bad contrast can still see what is active/selected.

This image showcases the new checkbox that appears on any selected alternative.
screen shot 2018-02-14 at 21 18 57

### CODE REVIEW

**Comment by sklirg** [comment]
@Letaguator I can't actually see the checkmark, did you forget to select an alternative for the screenshot? :-P

**Comment by duvholt** [comment]
IMO you shouldn't use float for this.
Float was created to allow content to flow around an element and this can cause various unexpected bugs. 

For example I think this looks a bit silly: 
image

I do however like the idea of adding a checkmark, but I think it should be implemented using something like flexbox instead. Vertical centering would also be nice.

**Comment by Letaguator** [comment]
@sklirg mixed up my screenshots :O

**Comment by Letaguator** [comment]
New screenshot, showing of changes:
screenshot_checkmark

File: client/src/components/Alternatives/Alternative/index.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ const Alternative = ({ disabled, id, text, selected, onClick }) => {
         />
         {text}
       </label>
+      <span
+        id="checked"
```
Comment (duvholt) [inline client/src/components/Alternatives/Alternative/index.js]: Why did you add this?

Other than that it looks good!
